### ISSUE TITLE
Replace <input> with <button>

### ISSUE THREAD

**Issue by mariannk** [nopSolutions/nopCommerce#423]
As time goes on and IE versions disappear one after another modernizing nopCommerce's markup becomes a very beneficial and very important task. One thing that is really easy to do and that would reap a lot of benefits is replacing all <input type="button/submit/reset" /> with <button type="button/submit/reset"></button>. I've described the reasons why in the list below:
1. <button> is absolutely the same as <input /> in the context of attributes, but slightly better. Name, Type, Disabled, Value are all standard, however with HTML5 the <button> received in addition SEVEN new attributes, six of which are specifically for forms and can provide useful functionality.
2. You can nest elements inside a <button>, and use the all so important and frequently used nowadays pseudo-elements! 
   < div >, < span >, < img / > are all valid elements to put inside of a <button>. This lets a Front-ender to be so much more creative (especially look at the buttons on the green and the purple background) and design more beautiful, complicated and sophisticated buttons. 
   A practical example: the only way to put an icon in an input is by setting it as background. As IconFonts become standard there would be no way to do that with input. However <button> has :before/:after to accomplish this and so much more.
3. Better cross-browser compatibility. < input / > buttons behave differently in different browsers i.e. line-height for < input / > buttons in Chrome and FF, and when you add iOS to the equation... <button> doesn't have these kind of problems which would save quite some time in the testing phase. (sadly FF has the ::-moz-focus-inner not only for <input>, but for <button> too, so don't forget to add a css reset for that)
   To summarize, <button> is the same as < input >... but much better :) 

Source: http://www.nopcommerce.com/boards/t/33253/upgrading-buttons.aspx
